README.md

```diff
@@ -531,18 +283,18 @@ Lavalink is a popular standalone audio sending node based on Lavaplayer. Lavalin
 and allows streaming music via many servers. It supports most of Lavaplayer's features.
 
 Lavalink is used by many large bots, as well as bot developers who can not use a Java library like Lavaplayer.
-If you plan on serving music on a smaller scale with JDA it is often preferable to just use Lavaplayer directly
+If you plan on serving music on a smaller scale with JDA, it is often preferable to just use Lavaplayer directly
 as it is easier.
 
 [Lavalink-Client](https://github.com/FredBoat/Lavalink-Client) is the official Lavalink client for JDA.
 
 
-### [jda-nas](https://github.com/sedmelluq/jda-nas) and [udpqueue](https://github.com/MinnDevelopment/udpqueue.rs)
+### [udpqueue](https://github.com/MinnDevelopment/udpqueue.rs) (an extension of [jda-nas](https://github.com/sedmelluq/jda-nas))
 
-Created and maintained by [sedmelluq](https://github.com/sedmelluq) and extended by [MinnDevelopment](https://github.com/MinnDevelopment)
-<br>Provides a native implementation for the JDA Audio Send-System to avoid GC pauses.
+Created and maintained by [sedmelluq](https://github.com/sedmelluq) and extended by [MinnDevelopment](https://github.com/MinnDevelopment)  
+Provides a native implementation for the JDA Audio Send-System to avoid GC pauses.
```

Comment:
```suggestion
Provides a native implementation for the JDA Audio Send-System **to avoid GC pauses potentially causing problems** with continuous audio playback.
```